code/modules/surgery/bodyparts/helpers.dm

```diff
@@ -215,3 +215,5 @@
 			. = "#fff4e6"
 		if("orange")
 			. = "#ffc905"
+		if("green")
+			. = "#A8E61D"
```

Comment:
all of the other hex codes are lowercase, i think this one should be too.

Reply:
also these should really all be defines like `SKINTONE_HEX_GREEN` or something instead of eating the cost on a proc override but that is a later project for later.

Reply:
Sir yes sir